docs/docs/cmd/spo/listitem/listitem-add.md

```diff
@@ -73,3 +73,9 @@ Add an item with a specific title to a list retrieved by server-relative URL in
 ```sh
 m365 spo listitem add --contentType Item --listUrl /sites/project-x/Documents --webUrl https://contoso.sharepoint.com/sites/project-x --Title "Demo Item"
 ```
+
+Add an item with a specific title and multi choice value
```

Comment:
Multi-choice is written with a hyphen. So 'multi-choice value' instead of 'multi choice value'